api/src/Repairers/Dto/CreateUserRepairerDto.php

```diff
@@ -71,4 +72,8 @@ final class CreateUserRepairerDto
     #[Groups([Repairer::REPAIRER_WRITE])]
     /** @var BikeType[] */
     public ?array $bikeTypesSupported = null;
+
+    #[Groups([Repairer::REPAIRER_WRITE])]
+    /** @var RepairerCity[] */
+    public ?array $repairerCities = null;
```

Comment:
```suggestion
    public ?array $repairerCities = [];
```

That's why a lot of your test fails, because `foreach` can't loop over a null value